pkg/codegen/internal/test/testdata/kubernetes-operator.pp.py

```diff
@@ -0,0 +1,169 @@
+import pulumi
+import pulumi_kubernetes as kubernetes
+
+pulumi_kubernetes_operator_deployment = kubernetes.apps.v1.Deployment("pulumi_kubernetes_operatorDeployment",
+    api_version="apps/v1",
+    kind="Deployment",
+    metadata={
+        "name": "pulumi-kubernetes-operator",
+    },
+    spec={
+        "replicas": 1,
+        "selector": {
+            "match_labels": {
+                "name": "pulumi-kubernetes-operator",
+            },
+        },
+        "template": {
+            "metadata": {
+                "labels": {
+                    "name": "pulumi-kubernetes-operator",
+                },
+            },
+            "spec": {
+                "service_account_name": "pulumi-kubernetes-operator",
+                "image_pull_secrets": [{
+                    "name": "pulumi-kubernetes-operator",
+                }],
+                "containers": [{
+                    "name": "pulumi-kubernetes-operator",
+                    "image": "pulumi/pulumi-kubernetes-operator:v0.0.2",
+                    "command": ["pulumi-kubernetes-operator"],
+                    "args": ["--zap-level=debug"],
+                    "image_pull_policy": "Always",
+                    "env": [
+                        {
+                            "name": "WATCH_NAMESPACE",
+                            "value_from": {
+                                "field_ref": {
+                                    "field_path": "metadata.namespace",
+                                },
+                            },
+                        },
+                        {
+                            "name": "POD_NAME",
+                            "value_from": {
+                                "field_ref": {
+                                    "field_path": "metadata.name",
+                                },
+                            },
+                        },
+                        {
+                            "name": "OPERATOR_NAME",
+                            "value": "pulumi-kubernetes-operator",
+                        },
+                    ],
+                }],
+            },
+        },
+    })
+pulumi_kubernetes_operator_role = kubernetes.rbac_authorization_k8s_io.v1.Role("pulumi_kubernetes_operatorRole",
```

Comment:
@komalali it looks like the python program gen is not consuming `moduleNameOverrides` in this instance